router/io-spec.gobra

```diff
@@ -469,25 +480,13 @@ pure func midSeg(
 
 ghost
 requires dp.Valid()
-requires len(raw) > 4
-requires acc(sl.AbsSlice_Bytes(raw, 0, len(raw)), _)
-requires unfolding acc(sl.AbsSlice_Bytes(raw, 0, len(raw)), _) in
-	let hdr := binary.BigEndian.Uint32(raw[0:4]) in
-	let metaHdr := scion.DecodedFrom(hdr) in
-	let seg1 := int(metaHdr.SegLen[0]) in
-	let seg2 := int(metaHdr.SegLen[1]) in
-	let seg3 := int(metaHdr.SegLen[2]) in
-	let base := scion.Base{metaHdr,
-		numInfoFields(seg1, seg2, seg3),
-		seg1 + seg2 + seg3} in
-	metaHdr.InBounds() &&
-	0 < metaHdr.SegLen[0] &&
-	base.ValidCurrInfSpec() &&
-	base.ValidCurrHfSpec() &&
-	len(raw) >= pktLen(seg1, seg2, seg3)
+requires acc(sl.AbsSlice_Bytes(raw, 0, len(raw)), R56)
+requires validPktMetaHdr(raw)
+opaque
```

Comment:
as a general guideline, we should put the `opaque` modifier in the line right after `ghost` to make it easier to read.

Reply:
Okay, I changed it.